presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionStringContextSuite.scala

```diff
@@ -0,0 +1,79 @@
+package dotty.tools.pc.tests.completion
+
+import dotty.tools.pc.base.BaseCompletionSuite
+import org.junit.Test
+import dotty.tools.dotc.core.Types.ThisType.raw
+
+class CompletionStringContextSuite extends BaseCompletionSuite:
+  @Test
+  def inScopeSymbol = check(
+    """
+    |object M:
+    | val VersionRegex = "".r
+    | VersionRe@@"1234"
+    """.stripMargin,
+    "|VersionRegex: Regex".stripMargin 
```

Comment:
Should we show VersionRegex if it doesn't have anything to do with StringContext? Could we have a check that only shows things like `raw` ?

Reply:
We want to show all completions. This is unintended behavior that there were no completions just before string due to String context desugaring. All other completions work like that

Reply:
for example you have a string which you want to wrap in Some, and you had no completions at all